### PR TITLE
Do not decrease compose hint font size

### LINKED ISSUE #4903: Placeholder size/position doesn't match actual entered text styling
The text box message placeholder's size/positioning doesn't match the text you actually see when typing. This leads to a somewhat jarring visual experience when looking at the app.

Here is the placeholder:

screenshot_20151216-195516 2

And here is "S" entered in the text box:

screenshot_20151216-195520 3

The placeholder is smaller than the actual entered S, so when you see the placeholder then start typing, the text feels like it suddenly changes size/style. It would be good if they lined up correctly. For example, here is Hangouts, where the "W" lines up exactly with the placeholder when entered.

screenshot_20151216-195454 3

screenshot_20151216-195448 3

With this, there's no discrepancy when beginning typing or deleting all entered text.

All screenshots here are taken at the same scaling, and are only cropped.

### PR DESCRIPTION
**Pull request by fehnomenal**
### First time contributor checklist

- [x] I have read [how to contribute](https://github.com/WhisperSystems/Signal-Android/blob/master/CONTRIBUTING.md) to this project
- [x] I have signed the [Contributor Licence Agreement](https://whispersystems.org/cla/)

### Contributor checklist

- [x] I am following the [Code Style Guidelines](https://github.com/WhisperSystems/Signal-Android/wiki/Code-Style-Guidelines)
- [x] I have tested my contribution on this device:
 * Nexus 5, Android 6.0
- [x] My contribution is fully baked and ready to be merged as is
- [x] I have made the choice whether I want the Bithub reward or not by omitting or adding the word `FREEBIE` in my commit message

----------

### Description

Fixes #4903

// FREEBIE

### CODE REVIEW

File: src/org/thoughtcrime/securesms/components/ComposeText.java [hunk 1]
```diff
@@ -58,12 +55,10 @@ private CharSequence ellipsizeToWidth(CharSequence text) {
...
 
   public void setHint(@NonNull String hint, @Nullable CharSequence subHint) {
-    this.hint = new SpannableString(hint);
-    this.hint.setSpan(new RelativeSizeSpan(0.8f), 0, hint.length(), Spannable.SPAN_INCLUSIVE_INCLUSIVE);
+    this.hint = hint;
 
     if (subHint != null) {
-      this.subHint = new SpannableString(subHint);
-      this.subHint.setSpan(new RelativeSizeSpan(0.8f), 0, subHint.length(), Spannable.SPAN_INCLUSIVE_INCLUSIVE);
+      this.subHint = subHint;
```
Comment (agrajaghh) [inline src/org/thoughtcrime/securesms/components/ComposeText.java]: if else isn't necessary anymore, no?

Comment (fehnomenal) [inline src/org/thoughtcrime/securesms/components/ComposeText.java]: Yeah, you're right. Removed it.

**Comment by moxie0** [comment]
I'll merge this, but I have a feeling we're going to find out that it was this way for a reason

**Comment by moxie0** [comment]
in 3.13

**Comment by cascheberg** [comment]
The feature (and font size) was originally introduced in #1337. Looks like it was just a design decision.

**Comment by aprescott** [comment]
Thanks for fixing this!

**Comment by 2-4601** [comment]
One downside is that in smaller screens (<4") the compose hint is now heavily truncated/ellipsized.

**Pull request closed without merging**